TYPO3.Flow/Tests/Functional/Resource/ResourceManagerTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace TYPO3\Flow\Tests\Functional\ResourceManagement;
```

Comment:
Should this maybe be `Functional\Resource` to make the autoloading failures go away?